docs/src/markdown-pages/aptitudes/filesystem.md

```diff
@@ -1,17 +1,31 @@
 ---
-name: "Filesystem"
-links_js: "Filesystem"
-description: "Provides read and write access to the filesystem, enabling Loop Authors to read and write files."
+name: 'Filesystem'
```

Comment:
@richseviora - prettier changed these up. It renders OK locally, do you think this will be an issue with Gatsby?

Reply:
Short answer - Not an issue
Long answer - Everything within that frontmatter section (between the `---` sections) is treated as YAML, and as you might know single quotes are also valid